vms/platformvm/txs/executor/state_changes.go

```diff
@@ -257,10 +251,97 @@ func advanceTimeTo(
 		changes.DeleteExpiry(expiry)
 	}
 
+	if !backend.Config.UpgradeConfig.IsEtnaActivated(newChainTime) {
```

Comment:
Can't comment on the line, but the link on line 231 should be updated to: https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#registerl1validatortx